src/main/app/controller/PostController.ts

```diff
@@ -78,6 +78,29 @@ export class PostController<T extends AnyObject> {
     });
   }
 
+  public async sendToApplicant2ForReview(req: AppRequest<T>, res: Response, formData: Partial<Case>): Promise<void> {
+    Object.assign(req.session.userCase, formData);
+    this.form.setFormState(req.session.userCase);
+    req.session.errors = this.form.getErrors(formData);
+
+    try {
+      const twoWeeks = 1000 * 60 * 60 * 24 * 14;
+      formData.dueDate = new Date(Date.now() + twoWeeks);
```

Comment:
I wonder if it would be best to set this field on the backend?

Reply:
Yeah I was weighing this up as well. Definitely happy to do it that way 👍